CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,77 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
```

Comment:
The CoC states the obvious. If we really need to have such things written, it should be thorough. I think something is missing, however. It does mention religion, but not ideology/belief/life stance/... (not sure which word to use in English; this is a bit outside my technical comfort zone). So it seems to exclude non-religious communists, atheists, humanists etc., as well as people with their own personal non-religious belief system that has not been labelled with a name. In the current version, this would make the CoC itself insultingly non-inclusive.

Reply:
I believe CoC are like software licenses. Imperfect. But better use an imperfection shared by a lot of people, with known limits that may be adressed in enhanced versions, that create our own imperfection... I don't really think we want to go in the business of writing our own CoC. At least not me :-)

Reply:
Yeah these things are hard to get right, the advantage of "adopting" the covenant code of conduct is that they make changes/improvements "upstream" so we do not get stuck trying to fix language on a project by project level.

Personally I tend to treat *harassment-free experience for everyone* as being inclusive and the rest of the sentence as a limited set of examples.

Would you like to raise a ticket here https://github.com/ContributorCovenant/contributor_covenant/issues ?